### PR TITLE
Query By Updated At

### PR DESCRIPTION
**Pull request by ShannonTalarico**

### CODE REVIEW

File: e2e/ChannelApeClient.spec.ts [hunk 1]
```diff
@@ -121,9 +121,9 @@ describe('ChannelApe Client', () => {
           return channelApeClient.actions().updateHealthCheck(expectedActionId).then((actualAction) => {
             throw new Error('Should not have succeeded');
           })
-          .catch((e) => {
-            expect(e.message).includes('Action has already been completed');
-          });
+            .catch((e) => {
```
Comment (ryankazokas) [inline e2e/ChannelApeClient.spec.ts:124]: @ShannonTalarico for ts files we are using 2 spaces for indentation.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "channelape-sdk",
-  "version": "1.24.0",
+  "version": "1.25.0-develop.0",
```
Comment (ryankazokas) [inline package.json:3]: This should be 1.26.0-develop.0

**Review by ryankazokas — CHANGES_REQUESTED** [review]
@ShannonTalarico let me know after you make these changes and i can merge.

File: package.json (hunk 2 above)
Comment (mb-ca) [reply package.json:3]: @ryankazokas There's already a 1.26.0-develop.0, and I just released 1.26.0 to master.  This will need to be 1.27.0

Comment (ryankazokas) [reply package.json:3]: Thanks @mb-ca . @ShannonTalarico sorry about that. Update version to 1.27.0-develop.0

Comment (ryankazokas) [reply package.json:3]: @ShannonTalarico i'll fix this during merge, but you should have updated the version in package.json and not package-lock.json. Package-lock.json is generated during npm install.

**Pull request merged**